src/utilities/typography.scss

```
@@ -208,6 +208,7 @@
 .ws-normal { white-space: normal !important; }
 
 /* Force long "words" to wrap if they exceed the width of the container */
+.wb-break-word,
 .break-word {
```

Maybe we can add a comment that `.break-word` is deprecated.

```suggestion
.break-word, // deprecated
.wb-break-word {
```